conftest.py

```diff
@@ -1,26 +1,40 @@
 import os
+
 from leapp.utils.repository import find_repository_basedir
 from leapp.repository.scan import find_and_scan_repositories
 
 
 def pytest_sessionstart(session):
     actor_path = os.environ.get('LEAPP_TESTED_ACTOR', None)
-    if not actor_path:
-        return
-    repo = find_and_scan_repositories(find_repository_basedir(actor_path), include_locals=True)
-    repo.load()
+    library_path = os.environ.get('LEAPP_TESTED_LIBRARY', None)
+
+    if actor_path:
+        repo = find_and_scan_repositories(find_repository_basedir(actor_path), include_locals=True)
+        repo.load()
 
-    actor = None
-    # find which actor is being tested
-    for a in repo.actors:
-        if a.full_path == actor_path.rstrip('/'):
-            actor = a
-            break
+        actor = None
+        # find which actor is being tested
+        for a in repo.actors:
+            if a.full_path == actor_path.rstrip('/'):
+                actor = a
+                break
 
-    if not actor:
-        return
+        if not actor:
+            return
 
-    # load actor context so libraries can be imported on module level
-    session.leapp_repository = repo
-    session.actor_context = actor.injected_context()
-    session.actor_context.__enter__()
+        # load actor context so libraries can be imported on module level
+        session.leapp_repository = repo
+        session.actor_context = actor.injected_context()
+        session.actor_context.__enter__()
+    elif library_path:
+        repo = find_and_scan_repositories(find_repository_basedir(library_path), include_locals=True)
+        repo.load()
+        # TODO: How to change directory to subpackage?
+        #       Consider that library_path directory contains package 'subpackage'
+        #       with tests in 'subpackage/tests' and that certain test needs
+        #       mocked file 'subpackage/tests/files/dummy'
+        #       When directory is changed to library_path, providing test with
+        #       the dummy file is only possible vit using 'full' path to the dummy
```

Comment:
I have no idea what you are saying here. I am trying to guess what it should have been:

```python
        # TODO: How to change the directory to a subpackage?
        #       Assuming that the library_path directory contains a package 'subpackage'
        #       with tests in 'subpackage/tests' and that certain tests need the mocked file
        #       'subpackage/tests/files/dummy'
        #       When the directory is changed to library_path, providing the test with the dummy
        #       file, is only possible by using the 'full' path to the dummy file -
        #       'subpackage/tests/files/dummy' instead of the 'relative' path 'tests/files/dummy'
```

Reply:
If this is what you meant, the answer is, that the test file should take care of injection of such things:
``` python
os.path.join(os.path.dirname(__file__), 'files', 'dummy')
```
And if it is about making it like it would be a repo/files then it would have to set / extend the environment variable `LEAPP_COMMON_FILES`

```python
# override an existing file in the repo with a mocked one:
override_path = os.path.join(os.path.dirname(__file__), 'files')
os.environ['LEAPP_COMMON_FILES'] =  ':'.join([override_path] + os.getenv('LEAPP_COMMON_FILES', '').split(':'))
```

Reply:
I updated the RTD section of repository layout with this info, see https://github.com/oamg/leapp/pull/532.